bootloader/src/rtl872x/rtl_it.c

```diff
@@ -128,6 +143,30 @@ void UsageFault_Handler(void) {
     Fault_Handler(UsageFault);
 }
 
+void SecureFault_Handler_NS(void) {
+    Fault_Handler(SecureFault);
+}
+
+__attribute__((always_inline)) inline
+void jump_to_nonsecure(u32 Addr) {
+    __ASM volatile ("MOV r0, %0\n\t"
+        "BLXNS   r0\n\t" : : "r" (Addr));
+}
+
+__attribute__((used, section(".secure.ram.text")))
```

Comment:
This line appears the key modification, it works for my test app as well.

@avtolstoy I was wondering why running the SecureFault_Handler in the secure RAM is not enough, we have to put it in the `.secure.ram.text`?